ql/src/java/org/apache/hadoop/hive/ql/io/parquet/LeafFilterFactory.java

```diff
@@ -37,6 +37,9 @@
 
 public class LeafFilterFactory {
   private static final Logger LOG = LoggerFactory.getLogger(LeafFilterFactory.class);
+  public static final String MESSAGE =
```

Comment:
Since it is public and used outside of this class, the name could be a bit more meaningful, like TYPE_NOT_SUPPORTED_MESSAGE, or sth similar. Also @VisibleForTesting annotation should be added to explicitly mark why it is public.

Another option to move the new tests in a separate test class placed in the org.apache.hadoop.hive.ql.io.parquet package. This would allow to reduce visibility to package private.

Reply:
Renamed variable to FILTER_PREDICATE_CONVERSION_NOT_SUPPORTED and added @VisibleForTesting